dashboards-observability/common/constants/shared.ts

```diff
@@ -124,4 +124,14 @@ export const LIVE_OPTIONS = [
   },
 ];
 
-export const LIVE_END_TIME ='now';
\ No newline at end of file
+export const LIVE_END_TIME ='now';
+
+export enum DefaultChartStyles {
```

Comment:
I think, this should be a class or interface rather than an enum. You should define an interface first with all these properties and can create a simple object of that interface type which have these default values. Enum is usually used when similar items are there for considerations. Something like ChartStyles, which can be "Bar", "Tree", "Gause"... But this is some characteristics you are defining here, whose properties are not similar in nature.

Reply:
Thanks, Subrat!
changes are done as per your suggestion.